grunt/config/replace.js

```diff
@@ -4,16 +4,12 @@ var _ = require('underscore');
 
 module.exports = function (grunt, options) {
 
-  var courseDir = path.join(options.sourcedir, 'course');
-
-  if (options.outputdir.indexOf('build') > -1) {
-    courseDir = path.join(options.outputdir, 'course');
-  }
-
-  var jsonext = grunt.config('jsonext');
-  var pathToConfig = path.join(courseDir, 'config.'+jsonext);
+  var courseDir = path.join(options.outputdir, 'course');
```

Comment:
switched to always use the build folder files as they're always available and it reduces the code here